test/functional/JLM_Tests/src/org/openj9/test/management/TestGetSystemCpuLoad.java

```diff
@@ -46,7 +46,7 @@
 	@BeforeMethod
 	protected void setUp() throws Exception {
 		String osName = System.getProperty("os.name");
-		if ((null == osName) || osName.equalsIgnoreCase("z/OS")) {
+		if (null == osName) {
```

Comment:
This whole method can be removed as it no longer serves any purpose - every platform should have an non-null `os.name`

Reply:
Change applied in ffdf65f8be493319c906b24c60b752782364cb35